addons/common/stringtable.xml

```diff
@@ -142,6 +153,7 @@
         </Key>
         <Key ID="STR_ZEN_Common_Sides">
             <English>Sides</English>
+            <French>Faces</French>
```

Comment:
Is this really the correct translation? Not a french myself but "Faces" doesn't sound like the "side" meant by BLUFOR / OPFOR / INDEPENDANT / CIVILIAN but more like the side of e.g. a cube.

Reply:
Oh okey! I'll change that!